### ISSUE TITLE
npm run demo is not working

### ISSUE THREAD

**Issue by darkrev613** [tabler/tabler-angular#20]
1. apps/demo/src/CNAME is missing.
2. after add a folder named CNAME then another error occurs.

ERROR in ./src/styles.scss (E:/Angular/tutorials/tabler-angular/node_modules/raw-loader!E:/Angular/tutorials/tabler-angular/node_modules/postcss-loader/lib??embedded!E:/Angular/tutorials/tabler-angular/node_modules/sass-loader/lib/loader.js??ref--14-3!./src/styles.scss)
Module build failed:
@import 'src/index.scss';
^
      File to import not found or unreadable: src/index.scss.
      in E:\Angular\tutorials\tabler-angular\libs\angular-styles\index.scss (line 1, column 1)
ERROR in apps\demo\src\app\app.module.ts(17,5): Error during template compile of 'AppModule'
  Could not resolve @tabler/angular-core relative to E:/Angular/tutorials/tabler-angular/apps/demo/src/app/app.module.ts..
apps/demo/src/app/app-routing.module.ts(3,54): error TS2307: Cannot find module '@tabler/angular-ui'.
apps/demo/src/app/app.service.ts(2,27): error TS2307: Cannot find module '@tabler/angular-ui'.
apps/demo/src/app/demo/demo.module.ts(2,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/app.module.ts(5,28): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/auth/auth.module.ts(3,30): error TS2307: Cannot find module '../../../../../libs/angular-core'.
apps/demo/src/app/components/components.module.ts(3,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/forms/components/forms-demo/forms-demo.component.ts(3,36): error TS2307: Cannot find module '@tabler/angular-forms'.
apps/demo/src/app/forms/forms.module.ts(3,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/gallery/gallery.module.ts(5,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/home/home.module.ts(4,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/interface/interface.module.ts(4,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/messages/messages.module.ts(4,30): error TS2307: Cannot find module '../../../../../libs/angular-core'.

**Comment by hamzicabdulah** [comment]
Hey. Have you managed to resolve these issues and run the app?

**Comment by arunabhdas** [comment]
Can you do the following 

==> npm install

==> npm run demo

Please also post your environment details to help me reproduce this issue.

**Comment by Max053** [comment]
Same problem for me. After remove CMAKE from the `angular.json` assets, I get the following error:


```
ERROR in ./src/styles.scss (/home/user/Projects/tabler-angular/node_modules/raw-loader!/home/user/Projects/tabler-angular/node_modules/postcss-loader/lib??embedded!/home/user/Projects/tabler-angular/node_modules/sass-loader/lib/loader.js??ref--14-3!./src/styles.scss)
Module build failed: 
@import 'src/index.scss';
^
      File to import not found or unreadable: src/index.scss.
      in /home/user/Projects/tabler-angular/libs/angular-styles/index.scss (line 1, column 1)
ERROR in src/app/app.module.ts(17,5): Error during template compile of 'AppModule'
  Could not resolve @tabler/angular-core relative to /home/user/Projects/tabler-angular/apps/demo/src/app/app.module.ts..
apps/demo/src/app/app-routing.module.ts(3,54): error TS2307: Cannot find module '@tabler/angular-ui'.
apps/demo/src/app/app.service.ts(2,27): error TS2307: Cannot find module '@tabler/angular-ui'.
apps/demo/src/app/demo/demo.module.ts(2,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/app.module.ts(5,28): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/auth/auth.module.ts(3,30): error TS2307: Cannot find module '../../../../../libs/angular-core'.
apps/demo/src/app/components/components.module.ts(3,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/forms/components/forms-demo/forms-demo.component.ts(3,36): error TS2307: Cannot find module '@tabler/angular-forms'.
apps/demo/src/app/forms/forms.module.ts(3,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/gallery/gallery.module.ts(5,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/home/home.module.ts(4,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/interface/interface.module.ts(4,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/messages/messages.module.ts(4,30): error TS2307: Cannot find module '../../../../../libs/angular-core'.
apps/demo/src/app/profile/profile.module.ts(3,30): error TS2307: Cannot find module '@tabler/angular-core'.
```

I don't know the project's structure as I just came across this project yesterday, but I've been looking into the source and I seem to be coming across a lot of errors.

The error I got seems to be because `libs/angular-styles/src/index.scss` seems to import `src/index.scss` while it's current location is in `src/lib/index.scss`.

When I fix this, I get the following error:
```
ERROR in src/app/app.module.ts(17,5): Error during template compile of 'AppModule'
  Could not resolve @tabler/angular-core relative to /home/max/Projects/tabler-angular/apps/demo/src/app/app.module.ts..
apps/demo/src/app/app-routing.module.ts(3,54): error TS2307: Cannot find module '@tabler/angular-ui'.
apps/demo/src/app/app.service.ts(2,27): error TS2307: Cannot find module '@tabler/angular-ui'.
apps/demo/src/app/demo/demo.module.ts(2,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/app.module.ts(5,28): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/auth/auth.module.ts(3,30): error TS2307: Cannot find module '../../../../../libs/angular-core'.
apps/demo/src/app/components/components.module.ts(3,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/forms/components/forms-demo/forms-demo.component.ts(3,36): error TS2307: Cannot find module '@tabler/angular-forms'.
apps/demo/src/app/forms/forms.module.ts(3,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/gallery/gallery.module.ts(5,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/home/home.module.ts(4,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/interface/interface.module.ts(4,30): error TS2307: Cannot find module '@tabler/angular-core'.
apps/demo/src/app/messages/messages.module.ts(4,30): error TS2307: Cannot find module '../../../../../libs/angular-core'.
apps/demo/src/app/profile/profile.module.ts(3,30): error TS2307: Cannot find module '@tabler/angular-core'.
```

I'm running KDE Neon which is based on Ubuntu 18.04. Nodejs v10.13.0, NPM v6.4.1. `ng --version` output:
```
Angular CLI: 6.0.1
Node: 10.13.0
OS: linux x64
Angular: 6.0.1
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.8
@angular-devkit/build-angular     0.6.8
@angular-devkit/build-optimizer   0.6.8
@angular-devkit/core              0.6.8
@angular-devkit/schematics        0.6.1
@ngtools/json-schema              1.1.0
@ngtools/webpack                  6.0.8
@schematics/angular               0.6.1
@schematics/update                0.6.1
ng-packagr                        2.4.5
rxjs                              6.0.0
typescript                        2.7.2
webpack                           4.8.3
```

**Comment by Max053** [comment]
I've forked this repository and I fixed the errors step by step. Most of the errors were false imports/references with wrong folder locations. It looked like someone either forgot to finish some work or someone messed up with merging. You can have a look at [my fork](https://github.com/Max053/tabler-angular) to see my fix. I have tested it on my Linux machine and will test it in a bit on a Windows 10 machine.

**Comment by arunabhdas** [comment]
Please try 0.7.7 and higher. This should be fixed in 0.7.7 and higher on both develop and master.
